src/homework19/PageObjects/Registration/RegistrationComponent.js

```diff
@@ -0,0 +1,38 @@
+export default class RegistrationComponent {
```

Comment:
RegistrationPopup or RegistrationPopupComponent

Reply:
Updated to RegistrationPopup